docs/widgets/last-donation.md

```diff
@@ -34,6 +47,7 @@ To update the settings update their value in `last-donation-settings.js` with an
 5. Click "OK" to create the source
 6. Check "Local file"
 7. Click "Browse" next to the "Local file" line and select `last-donation.html`
-8. Set "Width" to at least 300 (the widget will fill the whole width given)
-9. Set "Height" to 41 (+28 with header, +35 with recipiant)
-10. Click "OK" to finish the setup
+8. Set "Width" to at least 300 (the widget will fill the whole width given). 450 if using animation.
+9. Set "Height" to 41. 70 if using animation. (+28 with header, +35 with recipiant)
+10. Click "OK"
+11. Finally, if using animation, position the source to the right hand side of the screen since it animates from the right.
```

Comment:
Suggestion: How difficult would it be to make it left/right/top/bottom configurable?

Reply:
Left and right would be simple. Top and bottom would be more difficult. I will try them out.

Reply:
I added left and right options. Top and bottom are possible with little effort, but they are not simple to get the correct styles.